lib/estimator_rcs_impl.cc

```diff
@@ -192,7 +192,7 @@ void estimator_rcs_impl::handle_msg(pmt::pmt_t msg)
         }
     }
 
-    if (not(found_range && found_power))
+    if (!(found_range && found_power))
```

Comment:
@ryanvolz this is very interesting! I have to admit that my dev env is generally linux only, but I would have never suspected that this would yield to a build error in macOS and Windows :open_mouth: 

Do you know the reason behind that? I googled a bit (really a _short bit_) and didn't find something obvious

Reply:
I guess it's an MSVC thing. Something about maintaining compatibility with C code where "not" is a valid identifier.

Reply:
https://docs.microsoft.com/en-us/cpp/build/reference/permissive-standards-conformance?view=msvc-160&viewFallbackFrom=vs-2019 

The solution is to enable standards-compliant mode in MSVC, as far as I've been told. Would that be an option on your builds, @ryanvolz?

Reply:
I think that could be enabled for my builds, since it is supported with Visual Studio 2017. I imagine the easiest way to do that would be to pass the flag from CMake, as that's how I've done similar things before. I'm less sure if there are ways with conda-build to get the desired behavior from an environment variable or such.

Reply:
Hm, I mean: we can edit the CMake files in this repo, and ensure this also happens in GR upstream: For all other platforms, we use "standards-strict" mode! 

However, on maint-3.8, we only require MSVC>=1800, which I think is Visual Studio 2013 12.0. Don't know if that allows for `/permissive-`. On maint-3.9 and master, this should be no issue.

Reply:
Going by Microsoft's docs (and from what I'd expect knowing the standards compliance of their compilers), it really does require at least MSVC 2017 to make this work with the /permissive- flag. So that would be fine for 3.9 and master and works for me in practice, but as you say that would be an issue relative to historical support on 3.8.

Reply:
Do you know whether `#include <ciso646>` suffices on pre-2017 MSVC? That'd be something we could add to the MSVC-specific include headers in GR.

Reply:
You know what? Although it kind of hurts to change perfectly valid C++ for MSVC's pleasure, it **really** should not be holding this back!

Reply:
> Do you know whether `#include <ciso646>` suffices on pre-2017 MSVC? That'd be something we could add to the MSVC-specific include headers in GR.

I think that would work, perhaps best under an #ifdef on the C++ std since it will be going away for C++20. Moot point for this though. :smile: